dashboard/lib/pd/survey_pipeline/daily_survey_decorator.rb

```diff
@@ -53,5 +73,9 @@ def self.decorate(summary_data:, parsed_data:)
 
       result
     end
+
+    def self.get_form_name(form_id)
+      FORM_IDS_TO_NAMES[form_id]&.dig(:short_name) || form_id.to_s
```

Comment:
Use `dig` so it returns nil which is consistent with when a key doesn't exist in a hash. [] returns false instead.

Reply:
Worth a comment in code about how this function works?

Reply:
Found an even better way to do this in ruby `dig(first_key, second_key)`